### ISSUE TITLE
Added partial nl (Dutch) translation

### ISSUE THREAD

**Issue by hcderaad** [dmfs/opentasks#337]
I have translated your application partially into `nl` (*Dutch*). Here is the xml:
```xml
<!-- File "strings.xml" -->
<?xml version="1.0" encoding="utf-8"?>
<resources>

    <!-- Manifest stuff -->
    <string name="app_name">OpenTasks</string>
    <string name="permission_read_tasks">lees taken</string>
    <string name="permission_description_read_tasks">Staat toe een app om taken in je takenlijst te lezen.</string>
    <string name="permission_write_tasks">schrijf taken</string>
    <string name="permission_description_write_tasks">Staat toe een app om taken in je takenlijst aan te maken.</string>

    <!-- Task list -->
    <string name="menu_add_task">Voeg taak toe</string>
    <string name="menu_add_task_condensed">Voeg toe</string>
    <string name="menu_show_completed">Toon afgeronde taken</string>
    <string name="menu_hide_completed">Verberg afgeronde taken</string>
    <string name="menu_show_completed_condensed">Toon afgeronde taken</string>
    <string name="menu_sync_now">Ververs</string>
    <string name="menu_sync_now_condensed">Ververs</string>
    <string name="menu_search_hint">Zoeken</string>
    <string name="toast_task_completed">"%1s\" afgerond.</string>
    <string name="toast_task_uncompleted">"%1s\" heropend.</string>
    <string name="toast_x_removed">"%1s\" verwijderd.</string>
    <string name="task_group_title_list">Taken</string>
    <string name="task_group_title_start">Taken beginnen</string>
    <string name="task_group_title_due">Taken te doen</string>
    <string name="task_group_title_priority">Taak prioriteit</string>
    <string name="task_group_title_progress">Taak voortgang</string>
    <string name="task_group_title_default">Taken</string>
    <string name="quick_add_task_created">Taak toegevoegd</string>
    <string name="initial_local_task_list_name">Mijn taken</string>

    <plurals name="number_of_tasks">
        <item quantity="one">%d taak</item>
        <item quantity="other">%d taken</item>
    </plurals>

    <!-- Task detail view -->
    <string name="activity_task_details_edit">Bewerk</string>
    <string name="activity_task_details_edit_task">Bewerk taak</string>
    <string name="activity_task_details_delete">Verwijder</string>
    <string name="activity_task_details_delete_task">Verwijder taak</string>
    <string name="activity_task_details_complete">Rond af</string>
    <string name="activity_task_details_complete_task">Rond taak af</string>
    <string name="activity_task_details_pin">Pin</string>

    <!-- The title of the button to add a new item to a check list. -->
    <string name="opentasks_checklist_add_item">Item toevoegen</string>

    <!--
    The hint that's shown for a check list item that has no text.
    Should be something like "Item title", "Empty item", "new item" or "Enter item title", whatever works best in your language.
    -->
    <string name="opentasks_checklist_empty_item_hint">Item titel</string>

    <!-- Task editor -->
    <string name="activity_edit_task_title">Bewerk Taak</string>
    <string name="activity_add_task_title">Creëer Taak</string>
    <string name="activity_edit_task_error_invalid_url">Ongeldige URL</string>
    <string name="activity_edit_task_task_saved">Task opgeslagen.</string>
    <string name="activity_edit_task_empty_task_not_saved">Lege taak niet opgeslagen.</string>

    <!-- Names of task attributes -->

    <string name="task_list">Lijst</string>
    <string name="task_title">Titel</string>
    <string name="task_location">Locatie</string>
    <string name="task_description">Beschrijving</string>
    <string name="task_checklist">Check lijst</string>
    <string name="task_start">Start</string>
    <string name="task_due">Einde</string>
    <string name="task_completed">Afgerond</string>
    <string name="task_percent_complete">Percentage af</string>
    <string name="task_status">Status</string>
    <string name="task_classification">Privacy</string>
    <string name="task_url">URL</string>
    <string name="title_task_detail">Taakdetail</string>
    <string name="title_task_list">Taken</string>
    <string name="provider_label">Taken</string>
    <string name="task_detail_activity">Taakdetail</string>
    <string name="task_priority">Priority</string>
    <string name="task_timezone">Tijdzone</string>
    <string name="task_all_day">Hele Dag</string>

    <!-- Task list due group titles -->
    <string name="task_group_no_due">Geen einddatum</string>
    <string name="task_group_overdue">Over datum</string>
    <string name="task_group_due_today">Vandaag</string>
    <string name="task_group_due_tomorrow">Morgen</string>
    <string name="task_group_due_within_7_days">Komende dagen</string>
    <string name="task_group_due_in_month">Einde in %s</string>
    <string name="task_group_due_in_year">Einde in %s</string>
    <string name="task_group_due_in_future">Een keer</string>

    <!-- Task list start group titles -->
    <string name="task_group_start_started">Al Gestart</string>
    <string name="task_group_start_today">Vandaag</string>
    <string name="task_group_start_tomorrow">Morgen</string>
    <string name="task_group_start_within_7_days">Een dag</string>
    <string name="task_group_start_in_future">Later</string>
    <string name="task_group_due_title">Einde</string>
    <string name="task_group_all_title">Lijsten</string>
    <string name="task_group_completed_title">Afgerond</string>
    <string name="task_group_start_title">Start</string>
    <string name="task_group_search_title">Zoeken</string>
    <string name="task_group_priority_title">Prioriteit</string>
    <string name="task_group_progress_title">Voortgang</string>

    <!-- Task list progress group titles -->
    <string name="task_group_progress_0">Niets gedaan</string>
    <string name="task_group_progress_40">Goed begin</string>
    <string name="task_group_progress_60">Halverwege</string>
    <string name="task_group_progress_80">Bijna afgerond</string>
    <string name="task_group_progress_100">Afgerond</string>

    <!-- Task list priority group titles -->
    <string name="task_group_priority_high">Hoge prioriteit</string>
    <string name="task_group_priority_medium">Middelmatige prioriteit</string>
    <string name="task_group_priority_low">Lage prioriteit</string>
    <string name="task_group_priority_none">Geen prioriteit</string>

    <!-- Task editor labels -->

    <string name="action_save">Sla op</string>
    <string name="action_cancel">Annuleer</string>
    <string name="action_save_and_continue">Opslaan en doorgaan</string>

    <!-- Status labels -->

    <string name="status_needs_action">wacht op actie</string>
    <string name="status_in_process">in behandeling</string>
    <string name="status_completed">afgerond</string>
    <string name="status_cancelled">geannuleerd</string>
    <string name="status_incomplete">incompleet</string>

    <!-- Classification labels -->

    <string name="classification_not_specified">ongespecificeerd</string>
    <string name="classification_public">publiek</string>
    <string name="classification_private">privé</string>
    <string name="classification_confidential">vertrouwelijk</string>

    <!-- Priority labels -->

    <string name="priority_undefined">geen</string>
    <string name="priority_low">laag</string>
    <string name="priority_medium">middelmatig</string>
    <string name="priority_high">hoog</string>
    <string name="priority_urgent">urgent</string>

    <!-- Label for Settings -->

    <string name="title_activity_settings">Instellingen</string>
    <string name="action_settings">Instellingen</string>
    <string name="action_add_local_list">Lijst toevoegen</string>
    <string name="visible_task_lists">Getoonde Lijsten</string>
    <string name="enable_alarms">Notificaties</string>
    <string name="synced_task_lists">Gesynchroniseerde Lijst</string>

    <!-- dialogs -->
    <string name="task_list_settings_dialog_account_title">Kies een account</string>

    <!-- manage task list activity -->

    <string name="activity_manage_list_btn_save">Opslaan</string>
    <string name="activity_manage_list_btn_delete">Lijst verwijderen</string>
    <string name="activity_add_list_title">Lokale lijst aanmaken</string>
    <string name="activity_edit_list_title">Lijst instellingen</string>
    <string name="task_list_name_caption">Naam</string>
    <string name="task_list_no_sync">NB: Dit maakt een lokale lijst. Taken in een lokale lijst worden niet gesynchroniseerd.</string>
    <string name="task_list_color_caption">Lijst kleur</string>

    <!-- input dialog -->
    <string name="task_list_name_dialog_hint">Lijst naam</string>
    <string name="task_list_name_dialog_title">Voer een lijst naam in</string>
    <string name="task_list_name_dialog_error">Voer een naam in.</string>

    <!-- delete dialog -->
    <string name="task_list_delete_dialog_title">Verwijder %s</string>
    <string name="task_list_delete_dialog_message">Verwijder de lijst en alle taken in de lijst?\n
 NB: Er is geen mogelijkheid om een verwijderde lijst te herstellen.</string>
    <string name="task_list_delete_toast">%s verwijderd</string>

    <!-- Label for Expandable Group Descriptors -->
    <string name="gd_by_due_date">Per Einddatum</string>
    <string name="gd_by_completed">Per Status</string>
    <string name="gd_by_list">Per Lijst</string>

    <!-- String for confirm delete dialog -->
    <string name="confirm_delete_title">Bevestig verwijderen</string>
    <string name="confirm_delete_message">Verwijder deze taak?</string>
    <string name="confirm_delete_message_with_title">Verwijder \"%1s\"?</string>
    <string name="toast_task_deleted">"%1s\" verwijderd.</string>

    <!-- Strings in the Task List Selection Fragment -->
    <string name="task_list_selection_empty">Geen taak lijsten beschikbaar</string>
    <string name="task_list_selection_title">Kies lijsten om weer te geven</string>
    <string name="task_list_selection_ok">Ok</string>
    <string name="task_list_selection_cancel">Annuleren</string>

    <!-- Strings for Task List Widget -->
    <string name="task_list_widget_title">Huidige taken</string>
    <string name="task_list_widget_title_4x4">Huidige taken 4x4</string>

    <!-- Strings for fling actions -->
    <string name="fling_task_complete">Rond af</string>
    <string name="fling_task_edit">Bewerk</string>
    <string name="fling_task_delete">Verwijder</string>
    <string name="fling_task_uncomplete">Heropen</string>

    <!-- Accessibility content description -->
    <string name="cd_start_date">icoon voor startdatum van de taak</string>
    <string name="cd_due_date">icoon voor einddatum van de taak</string>

    <!-- Strings for notification -->
    <string name="notification_task_due_title">Taak einde</string>
    <string name="notification_task_start_title">Taak start</string>
    <string name="notification_task_due_date">Taak te doen, %1s</string>
    <string name="notification_task_start_date">Taak begint, %1s</string>
    <string name="notification_task_due_today">Taak vandaag te doen</string>
    <string name="notification_task_start_today">Taak begint vandaag</string>
    <string name="notification_action_completed">Afgerond</string>
    <string name="notification_action_delay_1h">+1 uur</string>
    <string name="notification_action_delay_1d">+1 dag</string>
    <string name="notification_action_delay_toast">Datum te doen bijgewerkt</string>
    <string name="notification_undo">Herstellen</string>
    <string name="notification_action_unpin">Ontpinnen</string>
    <string name="notification_task_pin_ticker">%1$s is gepind</string>

    <!-- Strings for time UI widgets -->
    <string name="button_add_one_hour">+1 uur</string>
    <string name="button_add_one_day">+1 dag</string>

    <!-- Dashclock -->
    <string name="dashclock_extension_title">Taken</string>
    <string name="dashclock_widget_title_due_expanded">%1$s moet worden gedaan - %2$s</string>
    <string name="dashclock_widget_title_due_expanded_allday">%1$s moet worden gedaan</string>
    <string name="dashclock_widget_title_start_expanded">%1$s begint - %2$s</string>
    <string name="dashclock_widget_title_start_expanded_allday">%1$s starts vandaag</string>
    <string name="dashclock_pref_displayed_task_summary">Selecteer taken en activiteiten om weer te geven in de dashclock widget</string>
    <string name="dashclock_pref_displayed_task_dialog_title">Activiteiten om weer te geven</string>
    <string name="dashclock_display_type_due">Taken te doen</string>
    <string name="dashclock_display_type_start">Beginnende taken</string>
    <string name="dashclock_display_type_start_and_due">Beginnende en huidige taken</string>
    <string name="dashclock_display_type_all">Alle</string>
    <string name="dashclock_display_type_pinned">Please\n
 Gepinde taken</string>
    <string name="dashclock_preference_title">Taak DashClock voorkeuren</string>

    <!-- General -->
    <string name="accept">Accepteren</string>
    <string name="now">Nu</string>

    <!-- Sharing intent -->
    <string name="share_as_task">Voeg taak toe</string>

</resources>

<!-- File "opentasks_defaults.xml" -->

<!-- File "dashclock.xml" -->
<?xml version="1.0" encoding="utf-8"?>
<resources>

    
    <string-array name="dashclock_displayed_task_types_values">
        <item name="1">1</item>
        <item name="2">2</item>
        <item name="3">3</item>
        <item name="4">4</item>
    </string-array>
    

</resources>

<!-- File "arrays.xml" -->


```

---

*Translation made with [Stringlate](https://github.com/LonamiWebs/Stringlate).*

**Comment by dmfs** [comment]
thanks @hcderaad 
@lemonboston please check and import the xml.

**Comment by dmfs** [comment]
merged, #356
